src/permissionGroups/index.tsx

```diff
@@ -40,7 +40,7 @@ interface PermissionGroupDetailsRouteProps {
 const PermissionGroupDetails: React.FC<
   RouteComponentProps<PermissionGroupDetailsRouteProps>
 > = ({ match }) => {
-  const qs = parseQs(location.search.substr(1));
+  const qs = parseQs(location.search.substr(1)) as any;
```

Comment:
**question**: why, is there some complex typing here?

Reply:
Previously `parseQs` return any by default but after the package upgrade they added more restricted typing